inspector/scripts/inspect-operators.js

```diff
@@ -107,17 +141,23 @@ module.exports = async function() {
             isUpToDateInTbtcPool = "N/A"
         }
 
+      const bondsAmountEth = web3.utils.fromWei(
+        web3.utils.toBN(totalOperatorBondsAmount[operator] || 0),
+      )
+
         ecdsaSummary.push({
             address: operator,
             eligibleStakeKeep: eligibleStakeKeep.toString(),
             operatorBalanceEth: operatorBalanceEth.toString(),
             unbondedValueEth: unbondedValueEth.toString(),
+        bondsAmountEth: bondsAmountEth.toString(),
```

Comment:
Can we add another column with a sum of bonded+unbonded?

Reply:
Yes, we can!